test/integration/singlecluster/controller/core/admissioncheck_controller_test.go

```diff
@@ -32,7 +32,7 @@ var _ = ginkgo.Describe("AdmissionCheck controller", ginkgo.Ordered, ginkgo.Cont
 	var ns *corev1.Namespace
 
 	ginkgo.BeforeAll(func() {
-		fwk.StartManager(ctx, cfg, managerSetup)
+		fwk.StartManager(ctx, cfg, managerAndControllerSetup(nil))
```

Comment:
Avoid these changes. I would suggest keeping `managerSetup` but just make it a wrapper for calling  managerAndControllerSetup(nil)

Reply:
Done